src/main/data/studios.json

```diff
@@ -1432,6 +1432,11 @@
     "code": "PSS",
     "description": "PRETTY SWEET"
   },
+  {
+    "code": "PTP",
+    "description": "Paper Tree Pictures, LLC",
+    "url": "https://cinedigm.com"
```

Comment:
@japierce1028 Can you confirm the URL?

Reply:
The entry into the google sheet registry shows this URL and the submitter has a Cinedigm.com email address. (The same person also submitted FAND on the same day). So I think the URL is good.